src/views/virtualmachines/list/VirtualMachinesList.tsx

```diff
@@ -91,48 +27,32 @@ type VMTableProps = {
 };
 
 const VMTable: React.FC<VMTableProps> = ({ data, unfilteredData, loaded, loadError, kind }) => {
```

Comment:
im guessing VMTable can be removed and VirtualizedTable<K8sResourceCommon> can be moved directly into ListPageBody?